src/main/java/org/jenkinsci/plugins/displayurlapi/DisplayURLProvider.java

```diff
@@ -98,7 +98,9 @@ public String getArtifactsURL(Run<?, ?> run) {
     /**
      * Fully qualified URL for a page that displays tests for a Run.
      */
-    public abstract String getTestsURL(Run<?, ?> run);
+    public String getTestsURL(Run<?, ?> run) {
```

Comment:
not sure if this is safe but otherwise we get an abstract method error because https://github.com/jenkinsci/blueocean-display-url-plugin/pull/24/files isn't released

could also catch the exception or maybe there's a better way?